### PR TITLE
jvmargs -> jvm options

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: tests/performance/basic/container/gcapp/services.xml [hunk 1]
```diff
@@ -5,7 +5,8 @@
   </admin>
 
   <container jetty="true" id="search" version="1.0">
-    <nodes jvmargs="-verbose:gc -Xms8g -Xmx8g -XX:+PrintGCDetails -XX:+UseG1GC ">
+    <nodes>
+      <jvm options="-verbose:gc -Xms8g -Xmx8g -XX:+PrintGCDetails -XX:+UseG1GC" />
```
Comment (baldersheim) [inline tests/performance/basic/container/gcapp/services.xml:9]: UseG1GC should be with gc-options

File: tests/search/bugs/4711376/app/services.xml [hunk 2]
```diff
@@ -10,7 +10,8 @@
 
   <container version="1.0">
     <nodes>
-      <node hostalias="node1" jvmargs="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005"/>
+      <jvm options="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005" />
```
Comment (baldersheim) [inline tests/search/bugs/4711376/app/services.xml:13]: Do we actually need debug options in a system test ?
I suspect that is options used for debugging locally that got committed by accident.

**Review by baldersheim — CHANGES_REQUESTED** [review]

File: tests/search/bugs/4711376/app/services.xml (hunk 2 above)
Comment (hmusum) [reply tests/search/bugs/4711376/app/services.xml]: Yeah, fixed

**Comment by hmusum** [comment]
@baldersheim please take a look again

**Review by baldersheim — APPROVED** [review]

**Pull request merged**